lib/shared/screens/admin/shared/components/options-list/option.jsx

```diff
@@ -101,15 +102,18 @@ export default class Option extends Component {
       OptionsList,
       value,
       white,
+      type,
```

Comment:
The string to boolean logic should be inside the Checkbox component. And you can verify if the value is a string and convert to boolean in case it is a string.